modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala

```diff
@@ -269,7 +269,7 @@ sealed abstract class Joining[F[_]: Async: GenUUID: SecurityProvider: KryoSerial
           Applicative[F].unit
         else ClusterIdDoesNotMatch.raiseError[F, Unit]
 
-      ownClusterSession <- clusterStorage.getClusterSession
+      ownClusterSession <- clusterStorage.getClusterToken
```

Comment:
I guess name should changed herę as well to be consistent

Reply:
It is consistent with `sessionStorage.getToken`

Reply:
Ewentually we can go for `clusterStorage.getToken`